pubspec.yaml

```diff
@@ -19,7 +19,7 @@ version: 1.0.0+1
 
 environment:
   sdk: '>=3.2.3 <4.0.0'
-  flutter: '>=3.3.0'
+  flutter: 3.19.6
```

Comment:
@atavism That is very cool, Now we don't have to change the version everytime on CI, It should take from pubspec.

Reply:
Yep, totally agree. I didn't realize we could do this